packages/@textlint/kernel/test/mocha.opts

```diff
@@ -1,2 +1,2 @@
 --recursive
---compilers js:babel-register
+--compilers ts:ts-node/register,js:babel-register
```

Comment:
Use `--require`.
Because Mocha 4 deprecated `--compilers` options.

I don't know that how to use mocha's `--require` and [ts-node](https://github.com/TypeStrong/ts-node)'s `--project` together.
So, I've used [ts-setup.js](https://github.com/textlint/textlint/blob/master/packages/textlint/test/ts-setup.js "ts-setup.js") as a workaround.

Reply:
I fixed it. Please check it again.